src/50_Samples_%26_Templates/Comment_Section.html

```diff
@@ -0,0 +1,142 @@
+    <script id="amp-access" type="application/json">
+    {
+        "authorization": "<%host%>/samples_templates/comment_section/authorization?rid=READER_ID&url=CANONICAL_URL&ref=DOCUMENT_REFERRER&_=RANDOM",
+        "noPingback": "true",
+        "login": {
+          "sign-in": "<%host%>/samples_templates/comment_section/login?rid=READER_ID&url=CANONICAL_URL",
+          "sign-out": "<%host%>/samples_templates/comment_section/logout"
+        },
+        "authorizationFallbackResponse": {
+            "error": true,
+            "loggedIn": false
+        }
+    }
+    </script>
+    </head>
+    <body>
+    <!-- #### Add a Comment Section -->
+    <h5 class="comments-heading">Comments</h5>   
+
+    <!-- Use `amp-list` to render existing comment to make sure that the latest comments appear if the page is loaded via AMP Cache.  --> 
+      <amp-list width=auto height=200 layout=fixed-height src="<%host%>/json/comments.json">
+        <template type="amp-mustache" id="amp-template-id">
+          <div class="comment-user comments-container">
+            <amp-img width="44" height="44" class="user-image" alt="user" src="/img/ic_account_box_black_48dp_1x.png"></amp-img>
+            <div class="card comment">
+              <p class= "date">{{user}} {{datetime}}</p>
+              <p>{{text}}</p>
+            </div>
+          </div>
+        </template>
+      </amp-list>
+
+    <!-- In order to add comments users need to be logged in. We use `amp-access` to integrate login. `on="tap:amp-access.login-sign-in"` specifies which 
+    action should be taken when clicking on the login button: `login` defines the property inside the `amp-access` json configuration, while `sign-in` defines
+    the endpoint. After clicking on the login button, a popup will appear; after clicking the displayed login button, the popup itself will disappear and the user will come back to sample page. We realize this behaviour by specifying an hidden input `return` with the value of [RETURN_URL](https://github.com/ampproject/amphtml/blob/master/extensions/amp-access/amp-access.md#access-url-variables) in the [login](https://github.com/ampproject/amp-by-example/tree/master/static/login.html) page; the AMP runtime will take care of the rest. --> 
+    <span amp-access="NOT loggedIn" role="button" tabindex="0" amp-access-hide>
+          <h5>Please login to comment</h5>
+          <a on="tap:amp-access.login-sign-in" class="button button-primary comment-button">Login</a>  
+    </span>    
+      <!-- Similar to the sign-in, the sign-out flow is specified by `on="tap:amp-access.login-sign-out"`. On the contrary of the sign-in, after clicking on the logout button the user stays on the same page: we realize this by redirecting to the URL specified by the `return` query parameter added by AMP runtime and adding `#success=true`. --> 
```

Comment:
same as above: " Similar to the sign-in, the sign-out flow is specified by `on="tap:amp-access.login-sign-out"`. On the contrary of the sign-in, after clicking on the logout button the user stays on the same page: we realize this by redirecting to the URL specified by the `return` query parameter added by AMP runtime and adding `#success=true`." is not needed here.